node/packages/parser/src/bitcoin/index.ts

```diff
@@ -8,15 +8,15 @@ import { toNetworkType } from './utils'
 
 export * from './types'
 
-export interface TransactionParserArgs {
```

Comment:
As these classes are all scoped within coin specific packages and we export them as named vars (ie. `export * as ethereum`), I intentionally did not prefix with the coin name again. I have always felt like this is redundant naming that we have through a lot of our stack (even though the majority of packages are separated by coin), but maybe that is not a shared opinion. Just wanted to provide a bit of context. Let me know your thoughts!

Reply:
That's useful context, thanks!

I don't think there is a right answer for this, though my thought process here is the change is a slight net improvement, even considering the redundancy/more verbose naming issues, which I agree with.

What I observe - although the Typescript compiler has no issues, IDEs can struggle in instances like this. As an example, when I use WebStorm to check the type of `TransactionParser` in https://github.com/shapeshift/unchained/blob/d6909f769cc943300ffeb7370e9f4a2b546a0da8/node/coinstacks/ethereum/ingester/src/workers/address.ts#L20, it actually resolves to `bitcoin`'s `TransactionParser`. 

I'm curious to know if you have the same issue in your IDE? If not, it could just be poor IDE configuration on my behalf, in which case we are muddying the codebase for no reason, and we should absolutely revert his change.

In summary, I agree - they are isolated modules that have no knowledge of each other, and _shouldn't_ need a domain prefix.

Reply:
As far as I can tell it's a WebStorm / JetBrain issue, where it'll resolve all types that match the name and return the first one.

In that case, I'll revert the changes in this commit, as we shouldn't compromise our code standards to work around a JetBrains bug.